terraform/modules/website-e2e-test/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "project_id" {
+  description = "Google Cloud Project to deploy module resources. (This is typically the 'ops' project.)"
+  type        = string
+}
+
+variable "region" {
+  type        = string
+  default     = "us-central1"
+  description = "Google Cloud Region"
+}
+
+variable "repo_owner" {
+  description = "Repo Owner (GoogleCloudPlatform)"
+  type        = string
+  default     = ""
+}
+
+variable "repo_name" {
+  description = "Repo Name (emblem)"
+  type        = string
+  default     = ""
```

Comment:
as a module, we can remove default values for `repo_name` and `repo_owner` since these won't be optional

Reply:
Done.